cmake/CMakejucer.cmake

```diff
@@ -81,4 +94,20 @@ function(jucer_project_end)
     ${JUCER_PROJECT_INCLUDE_DIRS}
   )
 
+  if(APPLE)
+    target_compile_options(${JUCER_PROJECT_NAME} PRIVATE -std=c++11)
+    target_compile_definitions(${JUCER_PROJECT_NAME} PRIVATE
+      $<$<CONFIG:Debug>:_DEBUG>
+      $<$<NOT:$<CONFIG:Debug>>:NDEBUG>
```

Comment:
This line should be removed, otherwise `NDEBUG` would be defined twice. @MartyLake can you confirm that `NDEBUG` is indeed defined twice currently? Thanks!